module/actuation/Kinematics/src/Kinematics.cpp

```diff
@@ -26,15 +27,37 @@ namespace module::actuation {
     using message::actuation::ServoState;
     using utility::actuation::kinematics::calculateHeadJoints;
     using utility::actuation::kinematics::calculateLegJoints;
+    using utility::actuation::tinyrobotics::configuration_to_servos;
+    using utility::actuation::tinyrobotics::servos_to_configuration;
     using utility::input::LimbID;
     using utility::input::ServoID;
 
     Kinematics::Kinematics(std::unique_ptr<NUClear::Environment> environment)
         : BehaviourReactor(std::move(environment)) {
 
-        on<Configuration>("Kinematics.yaml").then([this](const Configuration& cfg) {
+        on<Configuration>("Kinematics.yaml").then([this](const Configuration& config) {
             // Use configuration here from file Kinematics.yaml
-            this->log_level = cfg["log_level"].as<NUClear::LogLevel>();
+            this->log_level = config["log_level"].as<NUClear::LogLevel>();
+
+            // Parse NUgus URDF file description
+            cfg.urdf_path     = config["urdf_path"].as<std::string>();
+            nugus_model_left  = tinyrobotics::import_urdf<double, n_joints>(cfg.urdf_path);
+            nugus_model_right = tinyrobotics::import_urdf<double, n_joints>(cfg.urdf_path);
+
+            // Show the model if debug is enabled
+            if (log_level <= NUClear::DEBUG) {
+                nugus_model_left.show_details();
+            }
+
+            // tinyrobotics IK options
+            options.tolerance      = config["ik_tolerance"].as<double>();
+            options.max_iterations = config["ik_max_iterations"].as<int>();
+            options.method         = tinyrobotics::InverseKinematicsMethod::LEVENBERG_MARQUARDT;
```

Comment:
this seems like something we could put in config

Reply:
Yeah we could.